topics/web-components/index.md

```diff
@@ -0,0 +1,13 @@
+---
+aliases: webcomponent, web-component, webcomponents
+created_by: Alex Russell
+display_name: Web Components
+github_url: https://github.com/webcomponents
+logo: web-components.png
+related: custom-elements, shadow-dom, es-modules, html-templates, html-imports
+short_description: Web Components are a set of web platform APIs developers can use to create custom HTML tags.
+topic: web-components
+url: https://www.webcomponents.org/
+wikipedia_url: https://en.wikipedia.org/wiki/Web_Components
+---
+Developers can create custom, reusable, encapsulated HTML tags for the web. Components use the Custom Element standard and frequently are combined with the Shadow DOM, HTML Templates, and ES Modules.
```

Comment:
Maybe start the first sentence with "Web Components allow developers to create", or end with "for the web using Web Components."

Would https://developer.mozilla.org/en-US/docs/Web/Web_Components/Using_custom_elements be a good play to link "Custom Element standard" to?